src/main/java/ogc/rs/apiserver/util/AuthInfo.java

```diff
@@ -43,6 +44,27 @@ public static AuthInfo createUser(JsonObject tokenDetails) throws OgcException {
     return user;
   }
 
+  public static AuthInfo map(JsonObject tokenDetails){
```

Comment:
Rename this method to reflect that it's created using an authV2 or keycloak token

Reply:
done